server/src/modules/admin/admin.service.ts

```diff
@@ -185,4 +188,16 @@ export class AdminService {
 
         await this.userRepo.delete(userID);
     }
+
+    async getMaps(priority: boolean, status: MapStatus, skip: number, take: number, expand: string[]) {
+        // Where
+        const where: Prisma.MapWhereInput = {
+            status: status,
+            priority: priority
+        };
+
+        const dbResponse = await this.mapRepo.getAll(where, ExpandToPrismaIncludes(expand), null, skip, take);
+
+        return new PaginatedResponseDto(MapDto, dbResponse);
+    }
```

Comment:
I don't think we need this method. IMO the best approach is going to be to using the AdminCtlMapsGetAllQuery DTO, and MapsService.getAll can change to 
```ts
async getAll(userID: number, query: MapsCtlGetAllQuery | AdminCtlMapsGetAllQuery): Promise<PaginatedResponseDto<MapDto>> {
```
then add status and priority to the Prisma query if they're defined. Security-wise this should be fine, since class-validation will cause requests with bad queries to 400, so if a user calling GET maps/ included a status or priority, they'd 400.

Reply:
Worth writing an E2E test in maps.e2e-spec.ts to test that last bit!